staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -174,6 +174,22 @@ type Config struct {
 	// LoadBalancerResourceGroup determines the specific resource group of the load balancer user want to use, working
 	// with LoadBalancerName
 	LoadBalancerResourceGroup string `json:"loadBalancerResourceGroup,omitempty" yaml:"loadBalancerResourceGroup,omitempty"`
+
+	// AvailabilitySetNodesCacheTTL sets the Cache TTL for availabilitySetNodesCache
+	// if not set, will use default value
+	AvailabilitySetNodesCacheTTL int `json:"availabilitySetNodesCacheTTL,omitempty" yaml:"availabilitySetNodesCacheTTL,omitempty"`
```

Comment:
I think it's ambitious what is this int value. seconds? milliseconds? minutes?

maybe name these as "AvailabilitySetNodesCacheTTLInSeconds"

Reply:
Right, this is a good idea. Will change to it :)